old/bin/aa.pl

```diff
@@ -792,8 +792,8 @@ sub form_header {
       <th align=left>$form->{invtotal}</th>
      <td>| . (($form->{currency} ne $form->{defaultcurrency})
               ? $form->format_amount(\%myconfig,
-                                     $form->{invtotal}
-                                     * $form->{exchangerate}, 2) : '') . qq|</td>
+                                     $form->parse_amount( \%myconfig, $form->{invtotal} )
```

Comment:
Only a few lines up, `invtotal` gets formatted. Maybe we can save the unformatted value for this calculation instead of parsing it?

Reply:
We should indeed use formatted amount for output only and parse_amount for input only